OLSE.R

```diff
@@ -0,0 +1,17 @@
+z=read.csv("C:\\Users\\edaad\\Downloads\\dataunina.csv",header=T)
+library(tidyverse)
+
+head(z)
+for(i in 1:4){
+z[,i]=as.numeric(z[,i])
+}
+X=cbind(Const=rep(1,length(z[,1])),z[,2:4])
+for(i in 1:4){

```

Comment:
not sure why you are transforming X into a numeric again, I thought that's what you do on line 6. But I am not superfamiliar with R so I may be mistaken

Reply:
yeah, because I couldn't construct X'X. R is sometimes strange :)